spec/services/google_maps_service_spec.rb

```diff
@@ -10,7 +10,7 @@
       route_coordinates = gs.route_coordinates
 
       expect(route_coordinates).to         be_an(Array)
-      expect(route_coordinates.length).to  eq(41795)
+      expect(route_coordinates.length).to  eq(41797)
```

Comment:
Note to self to make this test less restrictive, this is the 2nd time we have needed to change this line due to new fixtures